ci-operator/step-registry/workers-rhel/repos/workers-rhel-repos-commands.sh

```diff
@@ -23,10 +23,15 @@ fi
 
 # Install an updated version of the client
 mkdir -p /tmp/client
-curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz | tar --directory=/tmp/client -xzf -
+curl https://mirror2.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz | tar --directory=/tmp/client -xzf -
```

Comment:
@jupierce do we really want mirror2.openshift.com? or just mirror.openshift.com?

Reply:
They are both aliases to the same backend and I intend to keep both available. i.e. use either.

Reply:
ah, good to know. Thx